src/log-filestore.c

```diff
@@ -484,6 +526,12 @@ static OutputCtx *LogFilestoreLogInitCtx(ConfNode *conf)
         SCLogInfo("forcing magic lookup for stored files");
     }
 
+    const char *write_meta = ConfNodeLookupChildValue(conf, "write-meta");
+    if (write_meta != NULL && !ConfValIsTrue(write_meta)) {
+        FileWriteMetaDisable();
+        SCLogInfo("Will not write meta file");
```

Comment:
this is not a great message in the default output mode. When function names are not printed there is no way for the user to know this is about filestore

Minor nitpick "files" instead "file"